frontend/src/pages/NewTeacher.vue

```diff
@@ -226,7 +229,7 @@ export default {
         NativeLanguageString: this.selected.NativeLanguage,
         SecondLanguageString: this.selected.SecondLanguage,
         EnglishProficiency: this.EnglishProficiency,
-        LanguageProficiency: this.SecondLanguageProficiency,
+        LanguageProficiency: this.NativeLanguageProficiency,
```

Comment:
Is there a reason you changed it from `SecondLanguageProficiency` to `NativeLanguageProficiency`? The way I understand it, a Teacher has the following properties:

- Native Language (with no proficiency, as you should be proficient in your native language)
- Second Language
- Second Language Proficiency
- English Proficiency (where even if both the Teacher's Native and Second languages are **not** English, we still store this)

Reply:
oh that makes sense. we were interpreting Native Language to have a proficiency. we'll revert this change.